kasa/smart/smartdevice.py

```diff
@@ -223,6 +237,7 @@ async def _initialize_features(self):
                     attribute_getter=lambda x: x._info["overheated"],
                     icon="mdi:heat-wave",
                     type=FeatureType.BinarySensor,
+                    category=Feature.Category.Debug,
```

Comment:
I'm wondering if we should have a separate category for warnings, or make them informational?